sbioapputils/app_runner/yaml_utils.py

```diff
@@ -0,0 +1,313 @@
+import json
+import os
+import yaml
+from .templates import csv_template, image_template, sc_template, default_template
+
+
+def get_yaml(workflow_loc):
+    """Helper function to parse the workflow configuration."""    
+    with open(workflow_loc, "r") as stream:
+        try:
+            yaml_dict = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+            
+    return yaml_dict
+
+
+def _define_files_from_yaml(yaml_dict):
+    
+    input_files = []
+    if yaml_dict.get('input_settings'):
```

Comment:
To decrease the nesting levels, you could do:
`if not yaml_dict.get('input_settings'):`
`    return`
in the beginning